src/Core/Users/UserLogin.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+namespace Meetup\Login\Core;
+
+use Meetup\Login\Core\Users\UserInterface;
+
+class UserLogin implements UserInterface {
+
+    /**
```

Comment:
Interfaces document methods. Classes don't need to. Please remove the `phpdoc` annotations.